ERROR: Cannot find PostgreSQL type 'POSTGRESQL_TYPE_BYTEA' in column type when process binary protocol value

## Bug Report

### Which version of ShardingSphere did you use?

master - 70c02a332868c971e34a326995bb8f9a100d5f35

This may affect all versions <= 5.3.1.

### Which project did you use? ShardingSphere-JDBC or ShardingSphere-Proxy?

ShardingSphere-Proxy PostgreSQL

### Expected behavior

```
ERROR: cannot cast type bytea to jsonb
```

### Actual behavior

Client log:
```
Exception in thread "main" org.postgresql.util.PSQLException: ERROR: Cannot find PostgreSQL type 'POSTGRESQL_TYPE_BYTEA' in column type when process binary protocol value
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2365)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:355)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:490)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:408)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:166)
	at org.postgresql.jdbc.PgPreparedStatement.executeUpdate(PgPreparedStatement.java:134)
```

Proxy log:
```
[ERROR] 2023-01-28 16:07:21.825 [Connection-17-ThreadExecutor] o.a.s.p.f.c.CommandExecutorTask - Exception occur: 
java.lang.IllegalArgumentException: Cannot find PostgreSQL type 'POSTGRESQL_TYPE_BYTEA' in column type when process binary protocol value
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:217)
	at org.apache.shardingsphere.db.protocol.postgresql.packet.command.query.extended.bind.protocol.PostgreSQLBinaryProtocolValueFactory.getBinaryProtocolValue(PostgreSQLBinaryProtocolValueFactory.java:145)
	at org.apache.shardingsphere.db.protocol.postgresql.packet.command.query.extended.bind.PostgreSQLComBindPacket.getBinaryParameters(PostgreSQLComBindPacket.java:140)
	at org.apache.shardingsphere.db.protocol.postgresql.packet.command.query.extended.bind.PostgreSQLComBindPacket.readParameters(PostgreSQLComBindPacket.java:81)
	at org.apache.shardingsphere.proxy.frontend.postgresql.command.query.extended.bind.PostgreSQLComBindExecutor.execute(PostgreSQLComBindExecutor.java:51)
	at org.apache.shardingsphere.proxy.frontend.postgresql.command.query.extended.PostgreSQLAggregatedCommandExecutor.execute(PostgreSQLAggregatedCommandExecutor.java:41)
	at org.apache.shardingsphere.proxy.frontend.command.CommandExecutorTask.executeCommand(CommandExecutorTask.java:110)
	at org.apache.shardingsphere.proxy.frontend.command.CommandExecutorTask.run(CommandExecutorTask.java:77)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
```

### Example codes for reproduce this issue (such as a github link).

```java
try (Statement statement = connection.createStatement()) {
    statement.executeUpdate("drop table if exists json_pg");
    statement.executeUpdate("create table json_pg (val_jsonb jsonb not null)");
}
try (PreparedStatement preparedStatement = connection.prepareStatement("insert into json_pg values (?)")) {
    preparedStatement.setBytes(1, "{\"test\":\"test\"}".getBytes(StandardCharsets.UTF_8));
    System.out.println(preparedStatement.executeUpdate());
}
```

--- a/db-protocol/postgresql/src/main/java/org/apache/shardingsphere/db/protocol/postgresql/packet/command/query/extended/bind/protocol/PostgreSQLBinaryProtocolValueFactory.java
+++ b/db-protocol/postgresql/src/main/java/org/apache/shardingsphere/db/protocol/postgresql/packet/command/query/extended/bind/protocol/PostgreSQLBinaryProtocolValueFactory.java
@@ -52,6 +52,7 @@ public final class PostgreSQLBinaryProtocolValueFactory {
         setFloat8ArrayBinaryProtocolValue();
         setBoolArrayBinaryProtocolValue();
         setStringArrayBinaryProtocolValue();
+        setByteaBinaryProtocolValue();
     }
     
     private static void setUnspecifiedBinaryProtocolValue() {
@@ -133,6 +134,10 @@ public final class PostgreSQLBinaryProtocolValueFactory {
         BINARY_PROTOCOL_VALUES.put(PostgreSQLColumnType.POSTGRESQL_TYPE_VARCHAR_ARRAY, new PostgreSQLStringArrayBinaryProtocolValue());
         BINARY_PROTOCOL_VALUES.put(PostgreSQLColumnType.POSTGRESQL_TYPE_CHAR_ARRAY, new PostgreSQLStringArrayBinaryProtocolValue());
         BINARY_PROTOCOL_VALUES.put(PostgreSQLColumnType.POSTGRESQL_TYPE_TEXT_ARRAY, new PostgreSQLStringArrayBinaryProtocolValue());
+    }
+    
+    private static void setByteaBinaryProtocolValue() {
+        BINARY_PROTOCOL_VALUES.put(PostgreSQLColumnType.POSTGRESQL_TYPE_BYTEA, new PostgreSQLByteaBinaryProtocolValue());
     }
     
     /**

--- a/db-protocol/postgresql/src/main/java/org/apache/shardingsphere/db/protocol/postgresql/packet/command/query/extended/bind/protocol/PostgreSQLByteaBinaryProtocolValue.java
+++ b/db-protocol/postgresql/src/main/java/org/apache/shardingsphere/db/protocol/postgresql/packet/command/query/extended/bind/protocol/PostgreSQLByteaBinaryProtocolValue.java
@@ -1,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.db.protocol.postgresql.packet.command.query.extended.bind.protocol;
+
+import org.apache.shardingsphere.db.protocol.postgresql.payload.PostgreSQLPacketPayload;
+
+/**
+ * Binary protocol value for bytea for PostgreSQL.
+ */
+public final class PostgreSQLByteaBinaryProtocolValue implements PostgreSQLBinaryProtocolValue {
+    
+    @Override
+    public int getColumnLength(final Object value) {
+        return ((byte[]) value).length;
+    }
+    
+    @Override
+    public Object read(final PostgreSQLPacketPayload payload, final int parameterValueLength) {
+        byte[] result = new byte[parameterValueLength];
+        payload.getByteBuf().readBytes(result);
+        return result;
+    }
+    
+    @Override
+    public void write(final PostgreSQLPacketPayload payload, final Object value) {
+        payload.writeBytes((byte[]) value);
+    }
+}

--- a/db-protocol/postgresql/src/test/java/org/apache/shardingsphere/db/protocol/postgresql/packet/command/query/extended/bind/protocol/PostgreSQLByteaBinaryProtocolValueTest.java
+++ b/db-protocol/postgresql/src/test/java/org/apache/shardingsphere/db/protocol/postgresql/packet/command/query/extended/bind/protocol/PostgreSQLByteaBinaryProtocolValueTest.java
@@ -1,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.db.protocol.postgresql.packet.command.query.extended.bind.protocol;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import org.apache.shardingsphere.db.protocol.postgresql.payload.PostgreSQLPacketPayload;
+import org.junit.Test;
+
+import java.nio.charset.StandardCharsets;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public final class PostgreSQLByteaBinaryProtocolValueTest {
+    
+    @Test
+    public void assertGetColumnLength() {
+        assertThat(new PostgreSQLByteaBinaryProtocolValue().getColumnLength(new byte[10]), is(10));
+    }
+    
+    @Test
+    public void assertRead() {
+        ByteBuf byteBuf = Unpooled.wrappedBuffer("value".getBytes(StandardCharsets.UTF_8));
+        PostgreSQLPacketPayload payload = new PostgreSQLPacketPayload(byteBuf, StandardCharsets.UTF_8);
+        assertThat(new PostgreSQLByteaBinaryProtocolValue().read(payload, 5), is("value".getBytes(StandardCharsets.UTF_8)));
+    }
+    
+    @Test
+    public void assertWrite() {
+        byte[] bytes = new byte[5];
+        ByteBuf byteBuf = Unpooled.wrappedBuffer(bytes).writerIndex(0);
+        PostgreSQLPacketPayload payload = new PostgreSQLPacketPayload(byteBuf, StandardCharsets.UTF_8);
+        byte[] expected = "value".getBytes(StandardCharsets.UTF_8);
+        new PostgreSQLByteaBinaryProtocolValue().write(payload, expected);
+        assertThat(bytes, is(expected));
+    }
+}

--- a/test/e2e/suite/src/test/java/org/apache/shardingsphere/test/e2e/cases/value/SQLValue.java
+++ b/test/e2e/suite/src/test/java/org/apache/shardingsphere/test/e2e/cases/value/SQLValue.java
@@ -20,6 +20,7 @@ package org.apache.shardingsphere.test.e2e.cases.value;
 import lombok.Getter;
 
 import java.math.BigDecimal;
+import java.nio.charset.StandardCharsets;
 import java.sql.Date;
 import java.sql.Time;
 import java.sql.Timestamp;
@@ -79,6 +80,8 @@ public final class SQLValue {
                 return new Time(new SimpleDateFormat("HH:mm:ss", Locale.getDefault()).parse(value).getTime());
             case "timestamp":
                 return new Timestamp(new SimpleDateFormat("yyyy-MM-dd HH:mm:ss", Locale.getDefault()).parse(value).getTime());
+            case "bytes":
+                return value.getBytes(StandardCharsets.UTF_8);
             default:
                 throw new UnsupportedOperationException(String.format("Cannot support type: `%s`", type));
         }
@@ -97,6 +100,9 @@ public final class SQLValue {
         }
         if (value instanceof Timestamp) {
             return formatString(new SimpleDateFormat("yyyy-MM-dd HH:mm:ss", Locale.getDefault()).format(value));
+        }
+        if (value instanceof byte[]) {
+            return formatString(new String((byte[]) value, StandardCharsets.UTF_8));
         }
         return value.toString();
     }

--- a/test/e2e/suite/src/test/java/org/apache/shardingsphere/test/e2e/engine/BatchE2EIT.java
+++ b/test/e2e/suite/src/test/java/org/apache/shardingsphere/test/e2e/engine/BatchE2EIT.java
@@ -33,6 +33,7 @@ import org.junit.After;
 import org.junit.Before;
 
 import javax.sql.DataSource;
+import java.nio.charset.StandardCharsets;
 import java.sql.Connection;
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
@@ -148,23 +149,25 @@ public abstract class BatchE2EIT extends BaseE2EIT {
         }
     }
     
-    private void assertRows(final ResultSet actualResultSet, final List<DataSetRow> expectedDatSetRows) throws SQLException {
+    private void assertRows(final ResultSet actual, final List<DataSetRow> expectedDatSetRows) throws SQLException {
         int count = 0;
-        while (actualResultSet.next()) {
-            int index = 1;
-            for (String each : expectedDatSetRows.get(count).splitValues(", ")) {
-                if (Types.DATE == actualResultSet.getMetaData().getColumnType(index)) {
-                    if (!NOT_VERIFY_FLAG.equals(each)) {
-                        assertThat(new SimpleDateFormat("yyyy-MM-dd").format(actualResultSet.getDate(index)), is(each));
+        while (actual.next()) {
+            int columnIndex = 1;
+            for (String expected : expectedDatSetRows.get(count).splitValues(", ")) {
+                if (Types.DATE == actual.getMetaData().getColumnType(columnIndex)) {
+                    if (!NOT_VERIFY_FLAG.equals(expected)) {
+                        assertThat(new SimpleDateFormat("yyyy-MM-dd").format(actual.getDate(columnIndex)), is(expected));
                     }
-                } else if (Types.CHAR == actualResultSet.getMetaData().getColumnType(index) && ("PostgreSQL".equals(getDatabaseType().getType()) || "openGauss".equals(getDatabaseType().getType()))) {
-                    assertThat(String.valueOf(actualResultSet.getObject(index)).trim(), is(each));
-                } else if (isPostgreSQLOrOpenGaussMoney(actualResultSet.getMetaData().getColumnTypeName(index))) {
-                    assertThat(actualResultSet.getString(index), is(each));
+                } else if (Types.CHAR == actual.getMetaData().getColumnType(columnIndex) && ("PostgreSQL".equals(getDatabaseType().getType()) || "openGauss".equals(getDatabaseType().getType()))) {
+                    assertThat(String.valueOf(actual.getObject(columnIndex)).trim(), is(expected));
+                } else if (isPostgreSQLOrOpenGaussMoney(actual.getMetaData().getColumnTypeName(columnIndex))) {
+                    assertThat(actual.getString(columnIndex), is(expected));
+                } else if (Types.BINARY == actual.getMetaData().getColumnType(columnIndex)) {
+                    assertThat(actual.getObject(columnIndex), is(expected.getBytes(StandardCharsets.UTF_8)));
                 } else {
-                    assertThat(String.valueOf(actualResultSet.getObject(index)), is(each));
+                    assertThat(String.valueOf(actual.getObject(columnIndex)), is(expected));
                 }
-                index++;
+                columnIndex++;
             }
             count++;
         }

--- a/test/e2e/suite/src/test/java/org/apache/shardingsphere/test/e2e/engine/dml/BaseDMLE2EIT.java
+++ b/test/e2e/suite/src/test/java/org/apache/shardingsphere/test/e2e/engine/dml/BaseDMLE2EIT.java
@@ -33,6 +33,7 @@ import org.junit.After;
 import org.junit.Before;
 
 import javax.sql.DataSource;
+import java.nio.charset.StandardCharsets;
 import java.sql.Connection;
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
@@ -129,6 +130,8 @@ public abstract class BaseDMLE2EIT extends SingleE2EIT {
             assertThat(String.valueOf(actual.getObject(columnIndex)).trim(), is(expected));
         } else if (isPostgreSQLOrOpenGaussMoney(actual.getMetaData().getColumnTypeName(columnIndex))) {
             assertThat(actual.getString(columnIndex), is(expected));
+        } else if (Types.BINARY == actual.getMetaData().getColumnType(columnIndex)) {
+            assertThat(actual.getObject(columnIndex), is(expected.getBytes(StandardCharsets.UTF_8)));
         } else {
             assertThat(String.valueOf(actual.getObject(columnIndex)), is(expected));
         }

--- a/test/e2e/suite/src/test/resources/cases/dml/dataset/passthrough/insert_bytea_values.xml
+++ b/test/e2e/suite/src/test/resources/cases/dml/dataset/passthrough/insert_bytea_values.xml
@@ -1,0 +1,24 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<dataset update-count="1">
+    <metadata data-nodes="passthrough.t_data_type_bytea">
+        <column name="id" type="numeric" />
+        <column name="val" type="varchar" />
+    </metadata>
+    <row data-node="passthrough.t_data_type_bytea" values="1, some values" />
+</dataset>

--- a/test/e2e/suite/src/test/resources/cases/dml/dml-integration-test-cases.xml
+++ b/test/e2e/suite/src/test/resources/cases/dml/dml-integration-test-cases.xml
@@ -333,4 +333,8 @@
     <test-case sql="UPDATE t_data_type_money SET val = val + ?::money WHERE id = ?" db-types="PostgreSQL,openGauss" scenario-types="passthrough">
         <assertion parameters="-12345.6789:String, 1002:int" expected-data-file="update_money_subtract_value.xml"/>
     </test-case>
+    
+    <test-case sql="INSERT INTO t_data_type_bytea (id, val) VALUES (?, ?)" db-types="PostgreSQL,openGauss" scenario-types="passthrough">
+        <assertion parameters="1:int, some values:bytes" expected-data-file="insert_bytea_values.xml"/>
+    </test-case>
 </integration-test-cases>

--- a/test/e2e/suite/src/test/resources/env/scenario/passthrough/data/actual/dataset.xml
+++ b/test/e2e/suite/src/test/resources/env/scenario/passthrough/data/actual/dataset.xml
@@ -45,6 +45,10 @@
         <column name="id" type="numeric" />
         <column name="val" type="cast#money" />
     </metadata>
+    <metadata data-nodes="passthrough.t_data_type_bytea">
+        <column name="id" type="numeric" />
+        <column name="val" type="varchar" />
+    </metadata>
     <row data-node="passthrough.t_data_type_integer_unsigned" values="1001, 18446744073709551615, 4294967295, 16777215, 65535, 255" />
     <row data-node="passthrough.t_data_type_integer_unsigned" values="1002, 0, 0, 0, 0, 0" />
     <row data-node="passthrough.t_data_type_money" values="1001, 123" />

--- a/test/e2e/suite/src/test/resources/env/scenario/passthrough/data/actual/init-sql/mysql/01-actual-init.sql
+++ b/test/e2e/suite/src/test/resources/env/scenario/passthrough/data/actual/init-sql/mysql/01-actual-init.sql
@@ -26,3 +26,4 @@ CREATE TABLE passthrough.t_data_type_integer_unsigned (id INT PRIMARY KEY, col_b
 CREATE TABLE passthrough.t_data_type_floating_point (id INT PRIMARY KEY, col_float REAL NOT NULL, col_double DOUBLE PRECISION NOT NULL);
 CREATE TABLE passthrough.t_with_generated_id (id INT AUTO_INCREMENT PRIMARY KEY, val VARCHAR(100) NOT NULL);
 CREATE TABLE passthrough.t_data_type_money (id INT PRIMARY KEY, val NUMERIC(16, 2));
+CREATE TABLE passthrough.t_data_type_bytea (id INT PRIMARY KEY, val BLOB NOT NULL);

--- a/test/e2e/suite/src/test/resources/env/scenario/passthrough/data/actual/init-sql/opengauss/01-actual-init.sql
+++ b/test/e2e/suite/src/test/resources/env/scenario/passthrough/data/actual/init-sql/opengauss/01-actual-init.sql
@@ -27,3 +27,4 @@ CREATE TABLE t_data_type_integer_unsigned (id INT PRIMARY KEY, col_bigint_unsign
 CREATE TABLE t_data_type_floating_point (id INT PRIMARY KEY, col_float REAL NOT NULL, col_double DOUBLE PRECISION NOT NULL);
 CREATE TABLE t_with_generated_id (id SERIAL PRIMARY KEY, val VARCHAR NOT NULL);
 CREATE TABLE t_data_type_money (id INT PRIMARY KEY, val money);
+CREATE TABLE t_data_type_bytea (id INT PRIMARY KEY, val bytea NOT NULL);

--- a/test/e2e/suite/src/test/resources/env/scenario/passthrough/data/actual/init-sql/postgresql/01-actual-init.sql
+++ b/test/e2e/suite/src/test/resources/env/scenario/passthrough/data/actual/init-sql/postgresql/01-actual-init.sql
@@ -27,3 +27,4 @@ CREATE TABLE t_data_type_integer_unsigned (id INT PRIMARY KEY, col_bigint_unsign
 CREATE TABLE t_data_type_floating_point (id INT PRIMARY KEY, col_float REAL NOT NULL, col_double DOUBLE PRECISION NOT NULL);
 CREATE TABLE t_with_generated_id (id SERIAL PRIMARY KEY, val VARCHAR NOT NULL);
 CREATE TABLE t_data_type_money (id INT PRIMARY KEY, val money);
+CREATE TABLE t_data_type_bytea (id INT PRIMARY KEY, val bytea NOT NULL);

--- a/test/e2e/suite/src/test/resources/env/scenario/passthrough/data/expected/dataset.xml
+++ b/test/e2e/suite/src/test/resources/env/scenario/passthrough/data/expected/dataset.xml
@@ -45,6 +45,10 @@
         <column name="id" type="numeric" />
         <column name="val" type="cast#money" />
     </metadata>
+    <metadata data-nodes="expected_dataset.t_data_type_bytea">
+        <column name="id" type="numeric" />
+        <column name="val" type="varchar" />
+    </metadata>
     <row data-node="expected_dataset.t_data_type_integer_unsigned" values="1001, 18446744073709551615, 4294967295, 16777215, 65535, 255" />
     <row data-node="expected_dataset.t_data_type_integer_unsigned" values="1002, 0, 0, 0, 0, 0" />
     <row data-node="expected_dataset.t_data_type_money" values="1001, 123" />

--- a/test/e2e/suite/src/test/resources/env/scenario/passthrough/data/expected/init-sql/mysql/01-expected-init.sql
+++ b/test/e2e/suite/src/test/resources/env/scenario/passthrough/data/expected/init-sql/mysql/01-expected-init.sql
@@ -26,3 +26,4 @@ CREATE TABLE expected_dataset.t_data_type_integer_unsigned (id INT PRIMARY KEY, 
 CREATE TABLE expected_dataset.t_data_type_floating_point (id INT PRIMARY KEY, col_float REAL NOT NULL, col_double DOUBLE PRECISION NOT NULL);
 CREATE TABLE expected_dataset.t_with_generated_id (id INT AUTO_INCREMENT PRIMARY KEY, val VARCHAR(100) NOT NULL);
 CREATE TABLE expected_dataset.t_data_type_money (id  INT PRIMARY KEY, val NUMERIC(16, 2));
+CREATE TABLE expected_dataset.t_data_type_bytea (id INT PRIMARY KEY, val BLOB NOT NULL);

--- a/test/e2e/suite/src/test/resources/env/scenario/passthrough/data/expected/init-sql/opengauss/01-expected-init.sql
+++ b/test/e2e/suite/src/test/resources/env/scenario/passthrough/data/expected/init-sql/opengauss/01-expected-init.sql
@@ -28,3 +28,4 @@ CREATE TABLE t_data_type_integer_unsigned (id INT PRIMARY KEY, col_bigint_unsign
 CREATE TABLE t_data_type_floating_point (id INT PRIMARY KEY, col_float REAL NOT NULL, col_double DOUBLE PRECISION NOT NULL);
 CREATE TABLE t_with_generated_id (id SERIAL PRIMARY KEY, val VARCHAR NOT NULL);
 CREATE TABLE t_data_type_money (id INT PRIMARY KEY, val money);
+CREATE TABLE t_data_type_bytea (id INT PRIMARY KEY, val bytea NOT NULL);

--- a/test/e2e/suite/src/test/resources/env/scenario/passthrough/data/expected/init-sql/postgresql/01-expected-init.sql
+++ b/test/e2e/suite/src/test/resources/env/scenario/passthrough/data/expected/init-sql/postgresql/01-expected-init.sql
@@ -28,3 +28,4 @@ CREATE TABLE t_data_type_integer_unsigned (id INT PRIMARY KEY, col_bigint_unsign
 CREATE TABLE t_data_type_floating_point (id INT PRIMARY KEY, col_float REAL NOT NULL, col_double DOUBLE PRECISION NOT NULL);
 CREATE TABLE t_with_generated_id (id SERIAL PRIMARY KEY, val VARCHAR NOT NULL);
 CREATE TABLE t_data_type_money (id INT PRIMARY KEY, val money);
+CREATE TABLE t_data_type_bytea (id INT PRIMARY KEY, val bytea NOT NULL);

